Makefile

```diff
@@ -151,16 +151,21 @@ clean-glide-cache:
 deps: prebuild-check
 	$(GLIDE_BIN) --verbose install
 
-.PHONY: generate
-## Generate GOA sources. Only necessary after clean of if changed `design` folder.
-generate: prebuild-check $(DESIGNS) $(GOAGEN_BIN) $(GO_BINDATA_ASSETFS_BIN) $(GO_BINDATA_BIN)
+app/controllers.go: $(DESIGNS) $(GOAGEN_BIN)
```

Comment:
They are outputs of the build step. That way Make can compare the modified dates on the target files vs the source files. If source > target, execute, else nothing to do.